File: src/main/java/com/rayco/domain/entity/EntryTypes.java
package com.rayco.domain.entity;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.SequenceGenerator;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Entity
@Getter
@Setter
@Table(name = "entry_types")
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class EntryTypes {

    @Id
    @GeneratedValue(strategy = GenerationType.SEQUENCE)
    @SequenceGenerator(sequenceName = "id_entry_types", name = "seq_generator")
    private Long id;

    @Column(name = "name", nullable = false)
    private String name;
}

File: src/main/kotlin/com/rayco/rayco_api/domain/entity/EntryTypes.kt
package com.rayco.rayco_api.domain.entity

import jakarta.persistence.Column
import jakarta.persistence.Entity
import jakarta.persistence.GeneratedValue
import jakarta.persistence.GenerationType
import jakarta.persistence.Id
import jakarta.persistence.SequenceGenerator
import jakarta.persistence.Table

@Entity
@Table(name = "entry_types")
data class EntryTypes(
    @Id
    @GeneratedValue(strategy = GenerationType.SEQUENCE)
    @SequenceGenerator(sequenceName = "id_entry_types", name = "seq_generator")
    val id: Long? = null,

    @Column(name = "name", nullable = false)
    val name: String
)